backend/src/main/java/com/praksa/auction/service/PaymentService.java

```diff
@@ -0,0 +1,40 @@
+package com.praksa.auction.service;
+
+import com.praksa.auction.dto.PaymentInfoDto;
+import com.praksa.auction.dto.PaymentResponseDto;
+import com.praksa.auction.model.Person;
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.model.PaymentIntent;
+import com.stripe.param.PaymentIntentCreateParams;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PaymentService {
+    @Autowired
+    PersonService personService;
+    @Value("${stripeSecretKey}")
+    private String apiKey;
+
+    public PaymentResponseDto createPaymentIntent(PaymentInfoDto paymentInfoDto) throws StripeException {
+        Person buyer = personService.getPersonById(paymentInfoDto.getCustomerId());
+        String custumerId = "";
```

Comment:
String custumerId = buyer.getCustomerId() ? personService.createCustomerId(buyer): buyer.getCustomerId();

Reply:
Inline if initialization is redundant

Reply:
How do you mean redundant ? It is Intelij message ?